docker-compose.yml

```diff
@@ -26,14 +26,14 @@ services:
       - "8500:8500"
       - "8501:8501"
     volumes:
-      - tf-data:/models/my_production_model/1/
+      - ./storage/models/my_production_model:/models/my_production_model/1/
 
   qdrant:
     image: generall/qdrant
     environment:
       RUST_BACKTRACE: "1"
     volumes:
-      - qdrant-data:/qdrant/storage
+      - ./storage/qdrant/storage:/qdrant/storage
```

Comment:
Alright, if it's more convenient for you, then feel free to remove the below `volumes` section ✌🏽